src/main/java/teammates/ui/controller/InstructorFeedbackPublishAction.java

```diff
@@ -24,7 +23,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         FeedbackSessionAttributes session = logic.getFeedbackSession(feedbackSessionName, courseId);
         boolean isCreatorOnly = false;
         
-        new GateKeeper().verifyAccessible(instructor, session, isCreatorOnly,
+        gateKeeper.verifyAccessible(instructor, session, isCreatorOnly,
                                           Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
```

Comment:
I've been abandoning "aligning for readability" recently precisely because of this: when you rename or change something, **it has a blast radius**, and sometimes a big one for that matter.

But I'll still fix the ones here in this PR.